ortools/linear_solver/xpress_interface.cc

```diff
@@ -2045,11 +2036,60 @@ MPSolver::ResultStatus XpressInterface::Solve(MPSolverParameters const& param) {
   return result_status_;
 }
 
+namespace {
+template <class T>
+struct getNameFlag;
+
+template <>
+struct getNameFlag<MPVariable> {
+  enum { value = XPRS_NAMES_COLUMN };
+};
+
+template <>
+struct getNameFlag<MPConstraint> {
+  enum { value = XPRS_NAMES_ROW };
+};
+
```

Comment:
Should will also do the same  for `MPObjective` ?
aka XPRS_NAMES_OBJECTIVE

Reply:
We could, but it seems unnecessary: we wouldn't use it, since MPObjective does not have a name